sklearn/tree/tree.py

```diff
@@ -300,6 +300,10 @@ def fit(self, X, y, sample_weight=None, check_input=True,
         elif self.presort == 'auto':
             presort = True
 
+        if type(presort) is not bool:
+            raise ValueError("'presort' should be either 'auto' or a boolean"
+                             " (True/False). Got {} instead.".format(presort))
```

Comment:
Same comment as https://github.com/scikit-learn/scikit-learn/pull/10110/files#r150681672.

Reply:
Oh! I dint understand what exactly was the format . The current error message format was suggested by @glemaitre. Could you elaborate little bit on the change be made? Thanks

Reply:
Instead of using a plain `{}`, use `{!r}` instead.